### DISCUSSION TITLE
Golden Soroban NFT Collection

### DISCUSSION THREAD

**Discussion by Mcmcav11** [stellar/sorobanathon#169] [General]
Here is my submission of NFT's for The Golden Soroban!

1) The Forest Temple
https://nft.fimi.market/detail/nft/Stellar/GBOJASOKEAYBI75QNMHZWXRNRS23JWGKP76KVAKJGNDO7TG2B7QWT75P/TFNU

2) Serpent Inferno
https://nft.fimi.market/detail/nft/Stellar/GB4Y2MRVDSFBTEKXBOAWRO5RFS2YUJFV36LK5VBR5735VURGZP4XLORA/TFNU

3) The Dangerous Path
https://nft.fimi.market/detail/nft/Stellar/GD7FE2MB3GZWSI2JJ54WU5P3BL2GHUDM4QGNYHGYG3G6MYOHUWN55IMO/TFNU

4) Cavern of Bones
https://nft.fimi.market/detail/nft/Stellar/GCOWPJI3N2CXNHXVL32JA2A4UQLRILBWONDOSSR4PB3ZHOUZLMOUBWM4/TFNU

5) The Dark Lake
https://nft.fimi.market/detail/nft/Stellar/GBSL76TACTMPVI2H5BT6IP4D6SPQWDNKMMGPIX5CAIL66I6UO5LRPJUU/TFNU

6) Good Luck Fish
https://nft.fimi.market/detail/nft/Stellar/GA74EOYDC2UE2GDX6IWO6CY2N6TAGH3ZGW5AU6X3ZL3HXT34ORSYPJB2/TFNU

7) Flash of The Soroban
https://nft.fimi.market/detail/nft/Stellar/GBHUA42VHNTYDRJMRFHOQRNVK732WPNWOKIYRCIAPAGJ2GFKMDYZ3SSP/TFNU

Thanks for the opportunity, I enjoyed it!
-Mike

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
